src/lib.rs

```diff
@@ -246,6 +317,23 @@ where
             ..self
         }
     }
+
+    /// Allow manual cache invalidation by setting the `X-Invalidate-Cache` header in the request.
+    /// This will allow the cache to be invalidated for the given key.
+    pub fn allow_invalidation(self) -> Self {
+        Self {
+            allow_invalidation: true,
+            ..self
+        }
+    }
+
+    /// Allow the response headers to be included in the cached response.
+    pub fn allow_response_headers(self) -> Self {
```

Comment:
Slight bikeshedding, but maybe rename to `add_response_headers(self)` (the method is not _allowing_ anything, it’s _setting_ the cache header on responses).